AdminUi/src/AdminUi/ClientApp/src/app/services/quotas-service/quotas.service.ts

```diff
@@ -10,10 +10,17 @@ import { environment } from "src/environments/environment";
 export class QuotasService {
     private readonly apiUrl: string;
 
+    private readonly refreshDataSubject = new BehaviorSubject<boolean>(false);
+    public refreshData$ = this.refreshDataSubject.asObservable();
```

Comment:
A service should always be stateless. Otherwise there may be weird side effects.